fem/quadinterpolator.hpp

```diff
@@ -128,8 +144,10 @@ class QuadratureInterpolator
 
    /// Template compute kernel for 3D.
    template<const int T_VDIM = 0, const int T_ND = 0, const int T_NQ = 0>
-   static void Eval3D(const int NE,
+   static void Mult3D(const int NE,
```

Comment:
Same here -- keep the old method as deprecated.

Reply:
Same here